### ISSUE TITLE
Adapt display ("pretty") name of the organisation …

### ISSUE THREAD

**Issue by Olf0** [sailfishos-chum/main#55]
… from "Sailfish OS Community Packages" to "SailfishOS:Chum community repository", see the upper left corner at [github.com/sailfishos-chum](https://github.com/sailfishos-chum).
Reason: That is the term we settled on and which was introduced in the past two days in various documents and at web-pages, specifically [the informational page this organisation links to](https://github.com/sailfishos-chum/main/blob/main/README.md).  Most importantly, the central word "Chum" is currently missing at the [organisation's frontpage](https://github.com/sailfishos-chum).

**Comment by rinigus** [comment]
What about "SailfishOS:Chum Community Packages" . Org is a collection of repositories, not just one

**Comment by rinigus** [comment]
Done.

**Comment by Olf0** [comment]
1. Sorry, I can usually spare some time at the weekend, but rarely during a working days.  Thus my reaction-time is usually *n* weeks; while I try to keep *n=1*, I often fail to (especially from spring to autumn).
2. To depict my intention for enhancing SailfishOS:Chum's documentation and appearance in general (not specific to this little issue here):
    * The bold goal of "[The ambition is to become the principal software distribution platform for Sailfish OS.](https://github.com/sailfishos-chum/main#sailfishoschum-community-repository-documentation-and-issue-tracker) has been set.
    * I do appreciate the scheme of building from the source a lot for transparency and security reasons, due to having experienced the positive effects of that at F-Droid for many years.
    * You do seem to have (at least some) support from sailors, which is unique (usually they harshly refrain from supporting anything which they do not own, AFAIK; e.g., see Halium) but well understandable due to the desolate state of their Store.  Thus I was positively surprised to see that some of them filed issues here (which they never did for OpenRepos, the Warehouse app or the Storeman app, AFAIR).
    * The appearance of many aspects of SailfishOS:Chum were in a typical "technicians state": Usable, but not nice at all in terms of appearance, language used, clarity etc.
      The line of thinking, that this is fully sufficient was nicely depicted by statements as "[*In practice, you just click at URL and focus on the next text rendered after clicking on it. I wouldn't worry about it …*](https://github.com/sailfishos-chum/main/pull/48#issuecomment-1013934095)" (meaning: the appearance does not matter as log as it is functional) or "[*Taking into account that these texts are mostly for developers and not of interest for users, I suggest to stop fine tuning them. I think they are good enough and its better to focus on more important issues*](https://github.com/sailfishos-chum/main/issues/45#issuecomment-1013659945)" (for a page to which multiple guides and a README direct potential users to).
    * While this is O.K. if technicians are the target group, this creates an impression which likely is fatal if users lacking a proper technical background are the target group: For them appearance is everything, because they do not comprehend the technical stuff at all, hence it is equivalent to magic for them.  I.e., they must judge the quality from the looks and feel, because they cannot use any other measure.
    * The consequences I have experienced so often are: Companies manage to sell crappy software as long as it does not immediately crash and looks nice, its dialogues have well formulated texts etc.; while Free Software which is functionally superior is denied due to its crappy appearance, texts in and for the software (e.g., documentation) which are apparently written by technicians for technicians (only) etc.
       Because it is mostly non-technical users using and judging a software (plus in a commercial context non-technical decision makers (i.e., managers) determining which software is deployed).
    * Hence slacking on appearance, quality and clarity of dialogues and documentation, the looks of logos and webpages etc. clearly contradicts the goal of "[The ambition is to become the principal software distribution platform for Sailfish OS".
       Though I do understand well that these aspects of a software and its documentation, webpages etc. are usually not on the radar of its developers, because they became software developers due to being interested in writing software, but not creating good looking webpages, documentation which addresses non-technical people, nice logos etc.
   * These aspects of software, the presentation, is something I have cared about every now and then for more than 15 years in my job environment (especially for Free Software), hence I assume to have an idea what the crucial aspects, the "quick wins" etc. are.
   * TL;DR: "I know what I am doing (… sort of :wink:)" or more seriously "I know that this is important to achieve, if the aim is to reach out beyond a purely technical audience".
3. Returning to this issue proper: Please do not stick too tightly to technical terms or the ones defined by GitHub!
    * "Organisation" is an awkward term GitHub has created for a multi-person umbrella for one or more git-repositories, because originally git-repositories only had a single "owner".
    * "Repository" is linguistically simply something into which one can put pieces of some sort and retrieve them later; i.e., this is far more loosely defined than a git-repository (for which the aforementioned pieces are git-commits) or a source code repository (for which the aforementioned pieces are source code files.)
    * IMO there is no reason which prevents the general term "repository" from being used recursively, i.e., the "pieces" could be (subordinate) "repositories", which is akin the situation here.  Still the top repository (of repositories) is a single one.
    * But most importantly (much more than all technical or linguistic details) one should adhere to a single, sounding, easily recognisable name for this, which is:<br />SailfishOS:Chum community repository
       As stated before: All technical or linguistic details are irrelevant for those who are not aware of them, which is the case for the vast majority of the target group (i.e., the average SailfishOS user).  But clearly denoting by consistently using the same name that these things belong together is crucial to enable simple users to realise that; otherwise they are driven to ponder if this is something different, related or completely separate etc. 
    * Originally I considered to call this (the "organisation" at GitHub) "SailfishOS:Chum source code repository" while the counterpart at the SailfishOS-OBS is the "SailfishOS:Chum packages repository", but this was me slipping into "technicians thinking" about technical details, which are irrelevant to transport to non-technicians!  As mentioned in the prior bullet point, most crucial thing is to have a single name.
    * I also considered to drop the "community" in "SailfishOS:Chum community repository" in order to shorten the name, but IMO the word "community" carries an important aspect and creates a warm feeling of togetherness.

Hence, taking all these consideration into account, I am asking you to name this endeavour consistently "SailfishOS:Chum community repository", both at GitHub (as the "organisation" name") and at the SailfishOS-OBS (where it technically is some intermediate repository, between the main OBS instance and the repositories of the individual software packages).

**Comment by Olf0** [comment]
> Hence, taking all these consideration into account, I am asking you to name this endeavour consistently "SailfishOS:Chum community repository", both at GitHub (as the "organisation" name") and at the SailfishOS-OBS (where it technically is some intermediate repository, between the main OBS instance and the repositories of the individual software packages).

Looking at the way a GitHub-user's or "organisation's" name is used in the SailfishOS:Chum-GUI app, I do see a value in shortening it and having it closer to a real organisation's name. Hence I alter my above suggestion for the name of this GitHub-"organisation" to:
**SailfishOS:Chum community**
As a pure subset of the name at SailfishOS-OBS ("SailfishOS:Chum community repository") this does not introduce any deviating words, which might create aforementioned confusion how this differs, and still carries all the other positive aspects detailed above.

**Comment by rinigus** [comment]
@Olf0 - thank you for your analysis and feedback! 

So, what would a suggestion? Do we go then for 

- SailfishOS:Chum community

or capitalized version

- SailfishOS:Chum Community

I would think that capitalized version is expected in this case. Reopening the issue

**Comment by Olf0** [comment]
I prefer the non capitalised "SailfishOS:Chum community", because that is the regular English spelling of "community".
IMO it is a hard to seriously argue that "community" is part of a name or a very specific term commonly used; only then it can be spelled with a capital "C" according to the English language rules.

But OTOH, this is in the realm of IT, where anything goes.  Still "SailfishOS:Chum community" looks much more natural to me.

**Comment by rinigus** [comment]
Well, for me it is a "title", hence capitalization question. @piggz, as a native speaker?

**Comment by Olf0** [comment]
Using "SailfishOS:Chum community" as a name (for the GitHub-organisation here or as displayed in the SailfishOS:Chum GUI app) does not make it being a name.

To be honest I fail to understand what "title" is supposed to address in this context, but for chapter titles in scientific publications the applied capitalisation rules seem to vary vastly between AE and BE as well as the specific author's preference.  But I do not think this is of relevance here, in contrast to (not really) being a name or not.

**Comment by rinigus** [comment]
OK. I don't mind setting it to "SailfishOS:Chum community". Done. If it is wrong for someone, we can always change it accordingly :)

**Comment by Olf0** [comment]
BTW and just for reference: Because this spelling "community" has been used or introduced at at couple of places, it would need to be altered consistently (e.g., when deciding for "Community"), see for example
* https://github.com/sailfishos-chum/main/blob/main/rpm/chum.spec#L1
* https://build.sailfishos.org/project/show/sailfishos:chum
